rpcs3/rpcs3qt/patch_manager_dialog.cpp

```diff
@@ -135,9 +136,25 @@ void patch_manager_dialog::load_patches()
 	path_list << QDir(QString::fromStdString(patches_path)).entryList(filters);
 	path_list.removeDuplicates(); // make sure to load patch.yml and imported_patch.yml only once
 
+	bool has_errors = false;
+
 	for (const auto& path : path_list)
 	{
-		patch_engine::load(m_map, patches_path + path.toStdString());
+		if (!patch_engine::load(m_map, patches_path + path.toStdString()))
+		{
+			has_errors = true;
+		}
+	}
+
+	if (show_error && has_errors)
+	{
+		// Open a warning dialog after the patch manager was opened
+		QTimer::singleShot(100, [this]()
```

Comment:
```suggestion
		QTimer::singleShot(100, [this, patches_path]()
```
I realized I can do it on my own xD